### PR TITLE
add support for rails 7

### PR DESCRIPTION
**Pull request by roman-melnyk**

### CODE REVIEW

**Comment by roman-melnyk** [comment]
@grosser hey! could you please review?

File: lib/soft_deletion/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module SoftDeletion
-  VERSION = '1.6.0'
+  VERSION = '1.7.0'
```
Comment (grosser) [inline lib/soft_deletion/version.rb:2]: should not bump in PRs

File: soft_deletion.gemspec [hunk 2]
```diff
@@ -9,7 +9,7 @@ Gem::Specification.new name, SoftDeletion::VERSION do |s|
   s.files = `git ls-files lib Readme.md`.split("\n")
   s.license = "MIT"
   s.required_ruby_version = '>= 2.5.0'
-  s.add_runtime_dependency 'activerecord', '>= 4.2.0', '< 6.2.0'
+  s.add_runtime_dependency 'activerecord', '>= 4.2.0', '< 7.1'
```
Comment (grosser) [inline soft_deletion.gemspec:12]: can do >=2.7 and >= 5.0.0

**Comment by grosser** [comment]
https://github.com/grosser/soft_deletion/pull/48

**Comment by grosser** [comment]
please rebase

**Comment by roman-melnyk** [comment]
done

**Comment by grosser** [comment]
run the `rake bundle_all` ... that should fix it 🤞

**Comment by grosser** [comment]
1.7.0

**Pull request merged**